src/assignments/11-higher-order-function.js

```diff
@@ -32,3 +32,20 @@ const users = [
 ];
 
 // Program 1: create a function and print the user details whose age>=30 (use filter and map)
+
+
+const usersOverThirty = (users) => {
+	users.map((item) => {
+		if (item.age >= 30){
+			item.id,
+			item.name,
+			item.age,
+			item.country
+			console.log(item);
+		}
```

Comment:
return the item here, so the new array would contains the user with age over or equal 30. 
`return item`